### ISSUE TITLE
Error while trying to upload Azure Bot written on Flask using https://docs.microsoft.com/en-in/azure/app-service/quickstart-python?tabs=powershell&pivots=python-framework-flask 

### ISSUE THREAD

**Issue by abhikdasgupta** [Azure/azure-cli#16706]
## Describe the bug

**Command Name**
`az webapp up`

**Errors:**
```
'NoneType' object has no attribute 'reserved'
Traceback (most recent call last):
Local\Temp\pip-unpacked-wheel-49qlvzn7\knack\cli.py, ln 215, in invoke
azure\cli\core\commands\__init__.py, ln 654, in execute
azure\cli\core\commands\__init__.py, ln 718, in _run_jobs_serially
azure\cli\core\commands\__init__.py, ln 709, in _run_job
azure\cli\command_modules\appservice\commands.py, ln 55, in _polish_bad_errors
azure\cli\core\commands\__init__.py, ln 688, in _run_job
azure\cli\core\commands\__init__.py, ln 325, in __call__
pip-unpacked-wheel-03qxfwck\azure\cli\core\__init__.py, ln 784, in default_command_handler
azure\cli\command_modules\appservice\custom.py, ln 3622, in webapp_up
AttributeError: 'NoneType' object has no attribute 'reserved'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az webapp up --sku {} --name {}`

## Environment Summary
```
Windows-10-10.0.18362-SP0
Python 3.6.8
Installer: MSI

azure-cli 2.14.0 *

```

**Comment by yonzhan** [comment]
webapp

**Comment by calvinsID** [comment]
This should be fixed in next release (2.19 or 2.18.1)
